rust/bridge/node/futures/src/util.rs

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2020 Signal Messenger, LLC.
+// SPDX-License-Identifier: AGPL-3.0-only
+//
+
+use std::any::Any;
+
+// See https://github.com/rust-lang/rfcs/issues/1389
+pub(crate) fn describe_any(any: &Box<dyn Any + Send>) -> String {
```

Comment:
Isn't this kind of specific to panics?  And only used for converting a panic to a JS error?  If so, why not just make a function that does that?

Reply:
It's not specific to JS errors, but it is specific to panics in practice. Would you prefer `describe_panic`?

Reply:
Yes, describe_panic seems better.